test/config.test.ts

```diff
@@ -38,58 +38,17 @@ describe('Config test', function () {
       .to.throw(/Could not read JSON config file at/)
   })
 
-  it.skip('should throw error if no serverUrl is set', function () {
+  it.skip('should throw error if no hackmdEndpointURL is set', function () {
```

Comment:
`hackmdEndpointURL` should be optional. If no value is given, the default endpoint `api.hackmd.io/v1` should be used.

Reply:
Fixed.